static/js/tools/choropleth/choropleth.tsx

```diff
@@ -72,7 +70,9 @@ class ChoroplethMap extends Component {
       //shouldComponentUpdate.
       this.renderGeoMap();
       this.updateGeoValues();
-    });
+    }, () => {
+      document.getElementById("heading").innerHTML = "API Request Failed!"
```

Comment:
whoa! that's a loud error... could you move these error messages to a different div that we can easily style later?

Reply:
also, error messages are generally more helpful when there's a recourse for the user. what should they do to get things working again?

Reply:
applies to the rest of the error messages as well

Reply:
I added better error messages for both of the UI and moved these over to a new div.